cmd/init.go

```diff
@@ -227,7 +227,7 @@ func askForDeployment(namespace string, c *kubernetes.Clientset) (*appsv1.Deploy
 	}
 	option, err := askForOptions(
 		options,
-		"Pick the deployment target for your okteto manifest from the list below:",
+		"What is the deployment target for your development container?",
```

Comment:
If this is going to be picked from a list, I'd use 'select' or 'pick' rather that 'What'. 
```suggestion
		"Select the deployment you want to replace with your development container:"
```